routes/catalog.js

```diff
@@ -0,0 +1,105 @@
+const express = require('express');
+
+const { db } = require('../server/db');
+
+const catalogRouter = express.Router();
+const { keysToCamel } = require('../common/utils');
+
+// -- GET - Returns all data from the catalog table
+catalogRouter.get('/', async (req, res) => {
+  try {
+    const allInfo = await db.query(`SELECT * from catalog;`);
+    res.status(200).json(keysToCamel(allInfo));
+  } catch (err) {
+    console.log(err);
+    res.status(500).send(err.message);
+  }
+});
+
+// -- GET/:id - Returns the row that matches the id
+// SELECT * FROM catalog WHERE id = ?;
+catalogRouter.get('/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const allUsers = await db.query(`SELECT * FROM catalog WHERE id = $1;`, [id]);
+    res.status(200).json(keysToCamel(allUsers));
+  } catch (err) {
+    console.log(err);
+    res.status(500).send(err.message);
+  }
+});
+
+// -- POST - Adds a new row to the catalog table
+// INSERT INTO catalog (id, host, title, event_type, subject, description, year)
+// VALUES (?, ?, ?, ?, ?, ?, ?);
+catalogRouter.post('/', async (req, res) => {
+  const { id, host, title, eventType, subject, description, year } = req.body;
+  try {
+    await db.query(
+      `INSERT INTO catalog (id, host, title, event_type, subject, description, year)
+      VALUES ($1, $2, $3, $4, $5, $6, $7);`,
+      [id, host, title, eventType, subject, description, year],
+    );
+    res.status(201).json({
+      status: 'Success',
+    });
+  } catch (err) {
+    res.status(500).json({
```

Comment:
Could you guys modify the output of the post request to also return the id of the row being modified? It'll help later with updating the front-end more efficiently

Reply:
I think returning both should be good!